scipy/odr/models.py

```diff
@@ -108,6 +108,27 @@ def polynomial(order):
     polynomial : Model instance
         Model instance.
 
+    Examples
+    --------
+    We can fit an input data using orthogonal distance regression (ODR) with
+    a polynomial model:
+
+    >>> import matplotlib.pyplot as plt
+    >>> from scipy.odr import odrpack as odr
```

Comment:
This import should be `from scipy.odr`. It has the same classes; `odrpack` is an implementation detail and wasn't meant to be public (it's just from before the "let's use underscores" era).

Reply:
Oh, I see. I fixed it.